src/ogg_vorbis.c

```diff
@@ -552,13 +552,13 @@ vorbis_rshort (SF_PRIVATE *psf, int samples, void *vptr, int off, int channels,
 		float inverse = 1.0 / psf->float_max ;
 		for (j = 0 ; j < samples ; j++)
 			for (n = 0 ; n < channels ; n++)
-				ptr [i++] = psf_lrintf ((pcm [n][j] * inverse) * 32767.0f) ;
+				ptr [i++] = CLAMP_SHORT (psf_lrintf ((pcm [n][j] * inverse) * 32767.0f)) ;
```

Comment:
Use of a macro will cause double evaluation of the expression - we don't want that.

Reply:
```suggestion
                float x = pcm [n][j] * inverse;
				ptr [i++] = psf_lrintf((x < -1 ? -1 : x > 1 ? 1 : x) * 32767.0f);
```